core/io/resource_loader.h

```diff
@@ -47,9 +47,11 @@ class ResourceFormatLoader : public RefCounted {
 
 public:
 	enum CacheMode {
-		CACHE_MODE_IGNORE, // Resource and subresources do not use path cache, no path is set into resource.
-		CACHE_MODE_REUSE, // Resource and subresources use patch cache, reuse existing loaded resources instead of loading from disk when available.
-		CACHE_MODE_REPLACE, // Resource and subresource use path cache, but replace existing loaded resources when available with information from disk.
+		CACHE_MODE_IGNORE,
```

Comment:
I believe something like this is doable and it would ensure we at least know there's some duplication when reading code.
```suggestion
		CACHE_MODE_IGNORE = ResourceLoader.CACHE_MODE_IGNORE,
```
I may be **completely** mistaken.

Reply:
That's a good point. However, I think that ship already sailed. This enum is not the only one duplicated across the source code. If we want to embrace an idiom like that to send a signal that this is just a mirror image of another (which I believe is a good idea), I'd suggest making a codebase-wide pass editing all cases. Also, that would be a good thing to make it easier to search for them to hopefully remove all duplicates next time we can break compat.